charts/values-overrides/examples/gke/README.md

```diff
@@ -166,12 +166,12 @@ helm repo add nvidia https://helm.ngc.nvidia.com/nvidia \
     && helm repo update
 ```
 
-Get the value of `nfd.enabled` using this command:
+To properly install the NVIDIA GPU Operator in Kubernetes, you must first check the value of the `nfd.enabled` label on your cluster nodes.  This label is used to determine whether Node Feature Discovery (NFD) is enabled, which is important because the GPU Operator depends on certain hardware features being correctly discovered.  Run the following command to retrieve the value of `nfd.enabled`:
```

Comment:
I think the user needs to install NFD before these commands will work, I am not sure if this is a pre-req or if it could be added to our helm charts.  @webcoderz maybe you can chime in here? But, here's some notes on installing NFD via kubectl or helm: 

```
# install NFD: 
	https://github.com/kubernetes-sigs/node-feature-discovery

	https://medium.com/@reefland/dynamic-kubernetes-node-labeling-via-node-feature-discovery-2c3e9c1879d1#:~:text=One%20of%20the%20best%20things,scan%20for%20on%20a%20node.


	was: 
		kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref=v0.12.2
	changed to this but used helm install install (see below): 
		kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref=v0.16.6


	export NFD_NS=node-feature-discovery

	helm repo add nfd https://kubernetes-sigs.github.io/node-feature-discovery/charts
	helm repo update
	# helm install nfd/node-feature-discovery -f <path/to/custom/values.yaml> --namespace $NFD_NS --create-namespace	
	helm install nfd/node-feature-discovery --namespace $NFD_NS --create-namespace	


	$  helm install nfd/node-feature-discovery --namespace $NFD_NS --create-namespace --generate-name
	NAME: node-feature-discovery-1734396303
	LAST DEPLOYED: Mon Dec 16 18:45:05 2024
	NAMESPACE: node-feature-discovery
	STATUS: deployed
	REVISION: 1
	TEST SUITE: None
```